sdk/formrecognizer/ai-form-recognizer/src/models.ts

```diff
@@ -345,6 +372,14 @@ export interface RecognizedForm {
    * Document type.
    */
   formType: string;
+  /**
+   * Confidence in the correctness of the form type.
+   */
+  formTypeConfidence?: number;
```

Comment:
if that name was not agreed on in archboard, perhaps you might want to make it more descriptive? e.g. `formTypeConfidenceScore` similar to what we do in TA?

Reply:
`formTypeConfidence` is what we presented to the Arch Board and is the current consensus, though this is something we could consider before GA. In the service it is called `docTypeConfidence` and I'm hesitant to get farther away from it.